artifactory/commands/transferconfig/transferconfig.go

```diff
@@ -160,8 +162,12 @@ func (tcc *TransferConfigCommand) runPreparations() error {
 	if err != nil {
 		return err
 	}
-	// Make sure source and target Artifactory URLs are different and the source Artifactory version is sufficient.
-	if _, err = tcc.ValidateMinVersionAndDifferentServers(); err != nil {
+	// Make sure that the source Artifactory version is sufficient.
+	if err = tcc.ValidateMinVersion(); err != nil {
+		return err
+	}
+	// Make sure source and target Artifactory URLs are different
+	if err = tcc.ValidateDifferentServers(); err != nil {
 		return err
 	}
 	// Make sure that the target Artifactory is empty and the config-import plugin is installed
```

Comment:
Code repetition here and in runPreChecks - let's bind ValidateMinVersion, ValidateDifferentServers, and validateTargetServer in a single method.